packages/ui-components/src/components/JsonEditor/JsonEditor.js

```diff
@@ -181,6 +181,10 @@ export class JsonEditor extends Component {
 
         const currentJson = this.jsonEditor.get();
         if (this.props.value !== currentJson) {
+          const validationError = await this.jsonEditor.validate();
```

Comment:
Does this work? I have a vague recollection that `this.jsonEditor.get()` throws an error if the json is invalid?

Reply:
Yes it works here. 
`.get()` validates json format and json schema if provided, but its error doesn't have schema error. I think JsonEditor wants to make schema error to be a warning instead of a major error.